src/main/java/io/vertx/ext/hawkular/impl/VertxMetricsFactoryImpl.java

```diff
@@ -37,4 +37,9 @@ public VertxMetrics metrics(Vertx vertx, VertxOptions vertxOptions) {
     }
     return new VertxMetricsImpl(vertx, vertxHawkularOptions);
   }
+
+  @Override
+  public MetricsOptions newOptions() {
```

Comment:
Yes, because it uses it to build the option object on which the setter methods are called.